src/main/java/org/jabref/gui/cleanup/CleanupAction.java

```diff
@@ -105,7 +105,7 @@ private void doCleanup(BibDatabaseContext databaseContext, CleanupPreferences pr
         CleanupWorker cleaner = new CleanupWorker(
                 databaseContext,
                 preferences.getFilePreferences(),
-                preferences.getTimestampPreferences());
+                preferences.getTimestampPreferences(), dialogService);
```

Comment:
Try to follow some code style around: one parameter for each line.

I know that IntelliJ formats differently, but you should review the generated code and adapt it. Nearly all diff tools show the lines around.